contract.php

```diff
@@ -269,3 +269,52 @@ function contract_civicrm_post($op, $objectName, $id, &$objectRef){
     }
   }
 }
+
+/**
+ * Adjust output of membership custom data for contract file download
+ * @param $content
+ * @param $context
+ * @param $tplName
+ * @param $object
+ */
+function contract_civicrm_alterContent(  &$content, $context, $tplName, &$object ) {
```

Comment:
Now implemented using jquery.  As there are no unique selectors for the custom data we have to rely on counting elements to modify the correct one.  So this may still break if the custom fields are rearranged - I can't think of a way around this!